gef.py

```diff
@@ -4567,6 +4567,7 @@ class PCustomCommand(GenericCommand):
 
     _cmdline_ = "pcustom"
     _syntax_  = "{:s} [-l] [StructA [0xADDRESS] [-e]]".format(_cmdline_)
+    _aliases_ = ["dt"]
```

Comment:
`dt` is an old alias, which is now only available by either adding manually or from the [gef-extras `windbg`](https://github.com/hugsy/gef-extras/blob/master/scripts/windbg.py) script.

Please do not re-add it. I will update the documentation to use the proper command (`pcustom`)